src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -16,19 +16,42 @@ redirects:
   - /docs/browser/new-relic-browser/installation/install-browser-monitoring-agent
 ---
 
-Browser uses a [JavaScript snippet](/docs/browser/new-relic-browser/page-load-timing/instrumentation-page-load-timing) (or "agent") to instrument your app's webpages. The JavaScript collects data for browser monitoring. To install the browser agent, you can choose from a number of deployment options.
+Browser uses a [JavaScript snippet](/docs/browser/new-relic-browser/page-load-timing/instrumentation-page-load-timing), also referred to as an "agent," to instrument your app's webpages. The JavaScript collects data for browser monitoring. To install the browser agent, you can choose from a number of deployment options.
 
 <Callout variant="tip">
   To use browser and the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
-## Select a deployment option [#options]
+## Enable browser monitoring [#enable]
 
-No matter which option you use to deploy browser monitoring, the end result is to inject the browser agent's JavaScript snippet into your pages for browser monitoring. The method you select depends on your preferences and business needs. To view the procedure to install and enable the browser agent, click the link for the option you want to use. You can also use the [Browser **Application settings** page](/docs/browser/new-relic-browser/configuration/browser-settings-ui-options-apdex-geography) to update settings.
+To get started enabling browser: Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**. The UI gives you instructions. 
```

Comment:
"To get started enabling" reads a little long for me. How about a more direct "To enable"?